src/audio/kp_detector.c

```diff
@@ -0,0 +1,28 @@
+#include <sof/audio/kp_detector.h>
+#include <sof/audio/component.h>
+
+void kp_dummy_func(void);
+struct comp_data dummy_cd;
+
+/* copy and process stream data from source to sink buffers */
+static int kp_detector_copy(struct comp_dev *dev)
```

Comment:
Seems to be a dummy test code, a final version should be part of kp-detector commit.

Reply:
Yes, I was thinking about merging this code with kp_detector once its PR is accepted.